### PR TITLE
[Doc] Declare expected outcomes

### PR DESCRIPTION
**Pull request by McSinyx**
Stabbing at GH-187.

### CODE REVIEW

File: docs/source/intro.rst [hunk 1]
```diff
@@ -86,28 +88,53 @@ students' overall progress to provide help if necessary.
...
+   Introduces the motivation and background of the problem
+   and some related works, and afterwards, define the objective
+   and scope of the project.
+
+Requirements
+   Specify use cases as well as non-functional requirements.
+
+Methodology
+   Illustrates the development steps, from technical choices
+   to architecture and system analysis and design.
```
Comment (9a24f0) [inline docs/source/intro.rst:130]: This on rendered pdf look not nice, please fix it, keep the bold if you could :smile:

**Review by 9a24f0 — CHANGES_REQUESTED** [review]

File: docs/source/intro.rst (hunk 1 above)
Comment (Huy-Ngo) [reply docs/source/intro.rst]: Seconded: It looks as if the bold words are a part of the hanging-indented paragraphs, whose second word are somehow capitalized.

image

File: docs/source/intro.rst [hunk 2]
```diff
@@ -86,28 +88,45 @@ students' overall progress to provide help if necessary.
...
+#. **Introduction**: Introduces the motivation and background of the problem
+   and some related works, and afterwards, define the objective
+   and scope of the project.
+#. **Requirements**: Specify use cases as well as non-functional requirements.
+#. **Methodology**: Illustrates the development steps, from technical choices
+   to architecture and system analysis and design.
+#. **Results and Discussion**: Show what we did in this project
+   and what we learned from it.
+#. **Conclusion and Future Work**: Summerize the findings
+   and suggest the work that can be done to improve the system.
```
Comment (Huy-Ngo) [inline docs/source/intro.rst:129]: Inconsistent conjugation: some verbs are conjugated as single and some as plural.

also s/Summerize/Summarize.

Comment (McSinyx) [reply docs/source/intro.rst:129]: It was to match the nouns on the left.

File: docs/source/intro.rst [hunk 3]
```diff
@@ -86,28 +88,44 @@ students' overall progress to provide help if necessary.
...
+#. Initial system design for both functional objectives
+   (:ref:`objcollab` and :ref:`objacademic`)
+#. Implementation of a subset of the use cases, focusing on :ref:`objcollab`,
+   while satisfying non-functional requirements to be ready for real-world usage
+
 Report Structure
 ----------------
 
-The report contains following sections:
+The report contains following chapters:
```
Comment (Huy-Ngo) [inline docs/source/intro.rst:117]: I think we are going to do sections now.

File: docs/source/intro.rst [hunk 4]
```diff
@@ -56,27 +56,29 @@ to study within a short period.
 Therefore, we started this project to create from scratch
 an academic-oriented project management system.
 
-Objective
----------
+Objectives
+----------
 
-In this project, we aimed to develop a collaboration platform
+In this project, the overall aims is to develop a collaboration platform
```
[the remark this answers is not in the document]
Comment (McSinyx) [reply docs/source/intro.rst:62]: Thanks, fixed.

File: docs/source/intro.rst (hunk 3 above)
Comment (McSinyx) [reply docs/source/intro.rst:117]: We will if possible, but at the moment they're still chapters.

File: docs/source/intro.rst (hunk 2 above)
Comment (Huy-Ngo) [reply docs/source/intro.rst]: They're just section names.

Comment (McSinyx) [reply docs/source/intro.rst]: Switched to infinitives.

**Review by 9a24f0 — APPROVED** [review]
:LGTM:

**Review by Huy-Ngo — APPROVED** [review]
OK I'll approve then

**Comment by McSinyx** [comment]
Thanks!

**Comment by Huy-Ngo** [comment]
> :LGTM:

Dear Unicode and GitHub can we have that.

**Comment by McSinyx** [comment]
How do you imagine the emoji to look like?

**Pull request merged**